packages/0xcert-ethereum-deploy-gateway-contracts/README.md

```diff
@@ -0,0 +1,15 @@
+<img src="https://github.com/0xcert/framework/raw/master/assets/cover-sub.png" />
+
+> Smart contracts used by the deploy gateway on the Ethereum blockchain.
+
+The [0xcert Framework](https://docs.0xcert.org) is a free and open-source JavaScript library that provides tools for building powerful decentralized applications. Please refer to the [official documentation](https://docs.0xcert.org) for more details.
+
+This module is one of the bricks of the [0xcert Framework](https://docs.0xcert.org). It's written with [TypeScript](https://www.typescriptlang.org) and it's actively maintained. The source code is available on [GitHub](https://github.com/0xcert/framework) where you can also find our [issue tracker](https://github.com/0xcert/framework/issues).
```

Comment:
This module is one of the bricks of the [0xcert Framework](https://docs.0xcert.org). The module is written with [TypeScript](https://www.typescriptlang.org) and is actively maintained. The source code is available on [GitHub](https://github.com/0xcert/framework) where you can also find our [issue tracker](https://github.com/0xcert/framework/issues).